include/hdmi_cec_driver.h

```diff
@@ -365,17 +356,17 @@ HDMI_CEC_STATUS HdmiCecSetTxCallback(int handle, HdmiCecTxCallback_t cbfunc, voi
 /**
  * @brief Synchronous transmit call
  *
- * This function writes a complete CEC packet onto the bus and waits for ACK.
+ * This function writes a complete CEC message onto the bus and waits for ACK.
  *
- * The packet contained in the buffer will follow the format detailed in HdmiCecSetRxCallback_t().
+ * The message contained in the buffer will follow the format detailed in HdmiCecSetRxCallback_t().
  * (ref <HDMI Specification 1-4> Section <CEC 6.1>)
  *
  *
  * @param[in] handle                              - The handle returned from the 
  *                                                    HdmiCecOpen() function. Non zero value
  * @param[in] buf                                 - The buffer contains a complete 
- *                                                    CEC packet to send.
- * @param[in] len                                 - Number of bytes in the packet.
+ *                                                    CEC message to send.
+ * @param[in] len                                 - Number of bytes in the message.
  * @param[out] result                             - send status buffer. Possible results are 
  *                    SENT_AND_ACKD,
```

Comment:
replace these 
SENT_AND_ACKD,
SENT_BUT_NOT_ACKD (e.g. no follower at the destination),
SENT_FAILED (e.g. collision).

Update these 
HDMI_CEC_IO_SENT_AND_ACKD,
HDMI_CEC_IO_SENT_BUT_NOT_ACKD (e.g. no follower at the destination),
HDMI_CEC_IO_SENT_FAILED (e.g. collision).

Reply:
updated